component/class/defaults.yml

```diff
@@ -11,8 +11,8 @@ parameters:
         source: https://charts.min.io
         version: 5.0.13
       keycloak:
-        source: https://charts.bitnami.com/bitnami
-        version: 18.3.0
+        source: https://codecentric.github.io/helm-charts
```

Comment:
is this their helm repository?

Reply:
No it's the codecentric one. They based their helm chart of the codecentric one. They just added a few defaults to the values.

We take those values and add them to our own values.

Inventage's repo is `https://docker-registry.inventage.com:10121`